test/unit/org/apache/cassandra/cql3/validation/operations/SelectSingleColumnRelationTest.java

```diff
@@ -456,6 +459,8 @@ public void testLargeClusteringINValues() throws Throwable
     @Test
     public void testMultiplePartitionKeyWithIndex() throws Throwable
     {
+        Assume.assumeFalse("SAI does not allow multi-column slice restrictions",
```

Comment:
Could be replaced with the new assume in Util + a comment?

Reply:
Done